src/main/resources/assets/js/components/NewAppModalComponent.jsx

```diff
@@ -259,9 +287,105 @@ var NewAppModalComponent = React.createClass({
                 errors={errors}>
               <input />
             </FormGroupComponent>
+            <hr />
```

Comment:
Don't use a hr-element for styling and separation. You could do separation by wrapping an element around. Line drawing could be done by CSS borders.